### PR TITLE
Remove duplicate code in PoolArena.

### PR DESCRIPTION
**Pull request by carryxyh**
Motivation:

Remove duplicate code in PoolArena.

Modification:

Replace duplicate code with `tinyIdx` and `smallIdx`.

Result:

Clean code.

### CODE REVIEW

**Comment by normanmaurer** [comment]
@netty-bot test this please

File: buffer/src/main/java/io/netty/buffer/PoolArena.java [hunk 1]
```diff
@@ -321,15 +321,10 @@ void freeChunk(PoolChunk<T> chunk, long handle, SizeClass sizeClass, ByteBuffer
         int tableIdx;
         PoolSubpage<T>[] table;
         if (isTiny(elemSize)) { // < 512
-            tableIdx = elemSize >>> 4;
+            tableIdx = tinyIdx(elemSize);
```
Comment (johnou) [inline buffer/src/main/java/io/netty/buffer/PoolArena.java:324]: @franz1981 guess these methods will be inlined?

Comment (normanmaurer) [reply buffer/src/main/java/io/netty/buffer/PoolArena.java:324]: yeah these are static and super hot so it should be inlined

**Comment by normanmaurer** [comment]
@carryxyh good catch

**Pull request merged**